README.md

```diff
@@ -225,6 +225,28 @@ Use the `task` method to configure tasks that are runnable via the CLI.  This al
 
 **Note**: only tasks configured using this method will be runnable from the CLI.
 
+##### `dry_tree_stub_cmd`, `dry_tree_stub_ssh`
```

Comment:
@jcredding we discussed renaming these to `stub_dry_tree_cmd` and `stub_dry_tree_ssh`.